tests/core/mempool/test_mempool_manager.py

```diff
@@ -101,9 +109,9 @@ def test_compute_assert_height() -> None:
     assert compute_assert_height(coin_records, conds) == 112
 
 
-def spend_bundle_from_conditions(conditions: List[List[Any]]) -> SpendBundle:
+def spend_bundle_from_conditions(coin: Coin, conditions: List[List[Any]]) -> SpendBundle:
```

Comment:
if you give `coin` a default value, you don't have to update every call site

Reply:
Done.